File: dev-superior-algoritmo-e-logica-de-programacao/cpp/exercicios/dentro_fora.cpp
#include <iostream>

using namespace std;

int main() {
    int N, X, fora, dentro;
    fora = 0;
    dentro = 0;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    for (int i = 0; i < N; i++) {
        cout << "Digite um numero: ";
        cin >> X;

        if ((X >= 10) && (X <= 20)) {
            dentro++;
        } else {
            fora++;
        }
    }

    cout << dentro << " DENTRO" << endl;
    cout << fora << " FORA" << endl;

    return 0;
}

File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/dentro_fora.py
N: int; X: int; fora: int; dentro: int

N = int(input("Quantos numeros voce vai digitar? "))

fora = 0
dentro = 0

for i in range(N):
	X = int(input("Digite um numero: "))

	if 10 <= X <= 20:
		dentro = dentro + 1
	else:
		fora = fora + 1

print(f"{dentro} DENTRO")
print(f"{fora} FORA")